### PR TITLE
feat: Accessibility and warnings

### LINKED ISSUE #34: Accessibility requirements during booking
Brief: Allow customers to provide details of accessibility requirements (such as limited mobility, wheelchair access, etc) when completing a booking

Requirements:
- Ability for submission of accessibility details (text field) on a booking model, which can be nullable
- On confirmation and payment of the booking, if accessibility details have been provided, email: 1) the contact_email associated with the production and 2) people who have the "view_bookings" permission on the production. We should email them details about their booking, along with the details that they provided us.

### PR DESCRIPTION
**Pull request by atoff**
Closes #34

TODO:

- [x] Implement other parts from https://github.com/BristolSTA/uobtheatre-api/issues/537
- [x] Email production staff when user provides accessibility info/requests

### CODE REVIEW

File: uobtheatre/bookings/models.py [hunk 1]
```diff
@@ -227,6 +227,8 @@ class Meta:
         default=0, validators=[MaxValueValidator(1), MinValueValidator(0)]
     )
 
+    assessibility_info = models.TextField(null=True, blank=True)
```
Comment (atoff) [inline uobtheatre/bookings/models.py:230]:
```suggestion
    accessibility _info = models.TextField(null=True, blank=True)
```

File: uobtheatre/bookings/emails.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    composer.send("Your booking is confirmed!", booking.user.email)
+
+
+def send_booking_with_accessibility_info_email(
+    booking: "Booking",
+):
+    """Sends an email to the production contact email, and to those with view sales and bookings permissions for a production, notifying them of a booking that has been amde with accessibility information"""
+
+    emails_to_notify = [booking.performance.production.contact_email] + list(
+        get_users_with_perm("productions.view_bookings", booking.performance.production)
```
Comment (JElgar) [inline uobtheatre/bookings/emails.py:65]: Im not a huge fan of this permission but guess its fine for now

File: uobtheatre/bookings/emails.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        .all()
+        .values_list("email", flat=True)
+    )
+
+    for email in emails_to_notify:
+        MailComposer().greeting().line(
```
Comment (JElgar) [inline uobtheatre/bookings/emails.py:71]: Can you construct the email once and then send it multiple times, rather that constructing it each time in the loop

**Review by JElgar — APPROVED** [review]
LGTM

**Pull request merged**